106: add: toggle turn off popup on mnemonic login (#129)
desmos-labs/mooncake: CHANGELOG.md

# Version 0.5.0
## Changes
- Improved how failed-to-sync posts are displayed
- Added the possibility of retry the sync or deleting failed-to-sync posts

# Version 0.4.3
## Bug fixes

> - Fixed mnemonic backup pop from displaying if user logged in using their mnemonic
> - Fixed Register flow that would allow user to return back to login

Reviewer: ```suggestion
- Fixed the mnemonic backup popup from displaying if the user logged in using their mnemonic backup
- Fixed the registration flow to allow the user to return back to the login page
```
